Fix the issue with the pre-processor test

Stefterv opened the pull request:
Upgrading the preprocessor made it no longer recognise the `quicktime` keyword in the `bug1532.pde` causing the test to fail. As far as I can see this test was never intended to test for that situation but rather for some invalid syntax later on line 50 of that file.

I cannot find anything about the `quicktime` keyword so I have just commented out the line with the new issue

Stefterv commented:
@SableRaf and I should do some archeological research into bug 1532 on the bugzilla archives

SableRaf commented:
@Stefterv Bug 1532 can be found at https://bugzilla.processing.org/1532

catilac commented on java/test/resources/bug1532.pde:
```diff
@@ -20,7 +20,7 @@ flatCube[][] grid;
 void setup() {
   try {
-    quicktime.QTSession.open();
+    // quicktime.QTSession.open();
```
Is there a reason for leaving the commented out code in? Or would it make sense to remove the whole `try { ... }` block?

Stefterv replied:
Nope, changed thanks!

Stefterv replied:
Oh yes there is, the line-count is load bearing

catilac replied:
@Stefterv oh is this because of a brittle test? or what do you mean?

Stefterv replied:
The test is looking for an error occurring on line 50, so by removing these lines it fails, an alternative would be to update the line count I guess

catilac reviewed (changes requested):
Thank you! Looks good, please remove the unused code and update the test

Stefterv committed:
Update bug1532 test and resource file

Adjusted expected exception position in ParserTests for bug1532 and removed QuickTime session initialization from bug1532.pde to reflect updated test requirements.

catilac reviewed (approved):
Thanks @Stefterv!
